app/matplotlib.ipynb

```diff
@@ -194,21 +257,33 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 10,
-   "outputs": [],
    "source": [
     "labels = [\"Product A\", \"Product B\", \"Product C\", \"Product D\"]\n",
     "market_share = [30, 20, 15, 35]\n",
-    "\n",
-    "# write your code here"
+    "plt.pie(market_share, labels=labels, autopct=\"%.1f%%\")\n",
+    "plt.title(\"Market share\")\n",
+    "plt.show()"
```

Comment:
In Task 5, you need to set the start angle for the pie chart for better view. You can do this by adding the `startangle` parameter in `plt.pie()`, e.g., `plt.pie(market_share, labels=labels, autopct="%.1f%%", startangle=90)`.